help/contributor/publishing.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Staging and publishing Knative documentation"
+linkTitle: "Website help"
+weight: "60"
+type: "docs"
+---
+
+## Testing your documentation
+
+Before you kick off a PR review you can verify that:
+
+- The page and any elements like widgets, titles, and links, work and
+  renders correctly.
+- If you moved or deleted a page, make sure to test that the redirects
+  (`aliases`) all work.
+
+### Auto PR preview builds
+
+All PRs are automatically built and a Preview is generated at:
+https://app.netlify.com/sites/knative/deploys
+
+This is currently a manual process.
+
+The build log is in the order of most recent to oldest. You must click to view
+the build log and then scroll down to see the `PR#`. When you find the build
+for your PR, click the [Preview] button to view the unique build preview for
+your PR. Each "event" in the PR triggers a new/unique build preview. Thefore,
+if you push a new commit, a new build with those changes will appear at the top
+of https://app.netlify.com/sites/knative/deploys.
```

Comment:
Maybe this section could be a procedure? Not a block on merging but just something to consider.

Reply:
Done